tests/test_summary.py

```diff
@@ -180,6 +180,32 @@ def test_res2csv_summary(tmp_path, mocker):
     assert str(disk_df["DATE"].values[-1]) == "2003-01-02"
 
 
+@pytest.mark.integration
+def test_res2csv_summary_no_output_on_invalid_input(tmp_path, mocker):
```

Comment:
This tests only tests for missing input, not invalid. That should be reflected in the test name (and then the comment below becomes superfluous)

Reply:
You are right. I have now added another test with empty input file to simulate invalid input.

Reply:
Good. I would have split it into two test functions as well.